surveys/2024-04/questions.md

```diff
@@ -0,0 +1,142 @@
+# Survey April 2024
+
+## Demographics
+
+- Where are you from? (countries list if possible - or open question)
+    - country
+
+- How long have you been working with Node.js? (numbers list from 0 to 10+)
```

Comment:
What unit? Years?

Reply:
We have a sliding scale in years in the survey

Reply:
@kyliewd The slider looks like its 0 to 100 instead of 0 to 10. I put it in the middle and its value is 50.

<img width="968" alt="image" src="https://github.com/nodejs/next-10/assets/229881/c957805c-e5cf-4438-8f65-eaf4f5d585a7">